tests/mapper/test_link_mappers.py

```diff
@@ -783,3 +783,390 @@ def test_Attri2VecLinkGenerator_unsupervisedSampler_sample_generation(self):
 
         with pytest.raises(IndexError):
             nf, nl = mapper[8]
+
+
+class Test_DirectedGraphSAGELinkGenerator:
+    """
+    Tests of GraphSAGELinkGenerator class
+    """
+
+    n_feat = 4
+    batch_size = 2
+    in_samples = [2, 4]
+    out_samples = [6, 8]
+
+    def test_LinkMapper_constructor(self):
```

Comment:
I think this could just be
```suggestion
    def test_constructor(self):
```